pkg/reportingplugins/mercury/types.go

```diff
@@ -41,3 +41,14 @@ type Transmitter interface {
 	// - FromAccount() should return CSA public key
 	ocrtypes.ContractTransmitter
 }
+
+type ChainReader interface {
+	// LatestBlocks returns an ordered list of the latest specified number of blocks
+	LatestBlocks(context.Context, int) ([]Head, error)
```

Comment:
Let's align the method name too:
```suggestion
	// LatestHeads returns an ordered list of the latest specified number of heads
	LatestHeads(context.Context, int) ([]Head, error)
```